emsurfer/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see https://editorconfig.org
```

Comment:
[red]
Can you move `.editorconfig` and `.gitignore` to the top of the repo please? Even though you're working on emsurfer only, these should be followed by all code.

Reply:
Done